tests/unit/storage/data/external/ExternalClients.js

```diff
@@ -154,6 +156,60 @@ describe('external backend clients', () => {
                     });
             });
         });
+
+        if (backend.config.type !== 'azure') {
+            it(`${backend.name} should set tags and then delete it`, done => {
+                const key = 'externalBackendTestKey';
+                const bucket = {
+                    getName: sinon.fake.returns('externalBackendTestBucket'),
+                };
+                const objectMd = {
+                    tags: {
+                        Key1: 'value_1',
+                        Key2: 'value_2',
+                    },
+                    location: [
+                        {
+                            dataStoreVersionId: 'latestversion',
+                        },
+                    ],
+                };
+                async.series([
+                    next => testClient.objectPutTagging(key.key, bucket, objectMd, log, next),
+                    next => testClient.objectDeleteTagging(key.Key, bucket, objectMd, log, next),
+                ], done);
+            });
+
+            it(`${backend.name} should fail to set tag on missing key`, done => {
+                const key = 'externalBackendMissingKey';
+                const bucket = {
```

Comment:
I would use the `BucketInfo` class here to make it closer to reality. https://github.com/scality/Arsenal/blob/development/8.1/lib/models/BucketInfo.js#L16
Might be using its static `fromObj` method?

Reply:
done.